### PR TITLE
README: Update heading and also maintainers

### PR DESCRIPTION
**Pull request by DhruvaG2000**
* New maintainers including Dhruva and Tokita, Hiroshi have been added.
* The title has been updated to specify what the project does at the top of README.

### CODE REVIEW

**Review by szczys — APPROVED** [review]
I suggest that @DhruvaG2000 be listed at the top of the Maintainers list, followed by @soburi as the two of you are doing all the maintenance and development work.

LGTM!

**Comment by beriberikix** [comment]
+1 adding them! Can we do it or do we need help from the Zephyr maintainers?

**Comment by alvarowolfx** [comment]
Any reason why I'm being removed ? I keep following the project but I haven't seen anything that I was requested to review. There are any meetings happening yet related to the project ?

**Comment by beriberikix** [comment]
Ah, I wasn't aware of that change. Can we add Alvaro back to the Maintainers?

File: README.md [hunk 1]
```diff
@@ -47,9 +45,8 @@ __NOTE:__ You can skip this step as well if you ran ``install.sh``.
 **Maintainers**:
 - [szczys](https://github.com/szczys)
 - [beriberikix](https://github.com/beriberikix) 
-- [alvarowolfx](https://github.com/alvarowolfx)
```
Comment (beriberikix) [inline README.md:50]: Add back

**Review by beriberikix — CHANGES_REQUESTED** [review]
All looks good but please add Alvaro back.

**Comment by soburi** [comment]
At the moment this repository is just a standalone repository, so I don't really care about the maintainer list.
But if we plan to put it under zephyr's governance, it's better to follow zephyr's rules.
zephyr clearly defines maintainers.


> Maintainer: lead Collaborators on an area identified by the TSC (e.g. Architecture, code subsystems, etc.). Maintainers shall also serve as the area’s representative on the TSC as needed. Maintainers may become voting members of the TSC under the guidelines stated in the project Charter.
> 
> Collaborator: A highly involved Contributor in one or more areas. May become a Maintainer with approval of existing TSC voting members.
> 
> Contributor: anyone in the community that contributes code or documentation to the project. Contributors may become Collaborators by approval of the existing Collaborators and Maintainers of the particular code base areas or subsystems.
> 

https://docs.zephyrproject.org/latest/project/project_roles.html#contributor

**Comment by beriberikix** [comment]
Makes sense! But we're not suggesting making those changes at this moment. Let's add you and update Dhruva's role in the README and decide if we want to change processes in a future PR :)

File: README.md (hunk 1 above)
Comment (DhruvaG2000) [reply README.md:50]: Pardon deleting @alvarowolfx , I have added him back

**Review by beriberikix — APPROVED** [review]
LGTM. Thank you for continuing to develop this module!

**Pull request merged**